yang/frr-bgp-route-map.yang

```diff
@@ -0,0 +1,855 @@
+          }
+        }
+
+        case peer-ipv6-address {
+          description
+            "IPv6 address of peer";
+          leaf peer-ipv6-address {
+            type inet:ipv6-address;
+          }
+        }
+
+        case peer-local {
+          description
+            "Static or Redistributed routes";
+          leaf peer-local {
+            type boolean;
+          }
+        }
+      }
+    }
+
+    case access-list-name {
+      when "derived-from-or-self(/frr-route-map:lib/frr-route-map:route-map/frr-route-map:entry/frr-route-map:match-condition/frr-route-map:condition, 'mac-address-list') orderived-from-or-self(/frr-route-map:lib/frr-route-map:route-map/frr-route-map:entry/frr-route-map:match-condition/frr-route-map:condition, 'as-path-list') orderived-from-or-self(/frr-route-map:lib/frr-route-map:route-map/frr-route-map:entry/frr-route-map:match-condition/frr-route-map:condition, 'ip-route-source') orderived-from-or-self(/frr-route-map:lib/frr-route-map:route-map/frr-route-map:entry/frr-route-map:match-condition/frr-route-map:condition, 'ip-route-source-prefix-list')";
+      description
+        "Access-list name";
+      leaf list-name {
+        type filter:access-list-name;
+      }
+    }
+
+    case evpn-default-route {
+      when "derived-from-or-self(/frr-route-map:lib/frr-route-map:route-map/frr-route-map:entry/frr-route-map:match-condition/frr-route-map:condition, 'evpn-default-route')";
+      description
+        "Match default EVPN type-5 route";
+      leaf evpn-default-route {
+        type empty;
+      }
+    }
+
+    case evpn-vni {
+      when "derived-from-or-self(/frr-route-map:lib/frr-route-map:route-map/frr-route-map:entry/frr-route-map:match-condition/frr-route-map:condition, 'evpn-vni')";
+      description
+        "Match eVPN VNI";
+      leaf evpn-vni {
+        type uint32 {
+          range "1..16777215";
+        }
+      }
+    }
+
+    case evpn-route-type {
+      when "derived-from-or-self(/frr-route-map:lib/frr-route-map:route-map/frr-route-map:entry/frr-route-map:match-condition/frr-route-map:condition, 'evpn-route-type')";
+      description
+        "Match eVPN route-type";
+      leaf evpn-route-type {
+        type enumeration {
+          enum "macip" {
+            value 0;
+            description
+              "Mac-IP route";
+          }
+          enum "multicast" {
+            value 1;
+            description
+              "IMET route";
+          }
+          enum "prefix" {
+            value 2;
+            description
+              "Prefix route";
+          }
+        }
+      }
+    }
+
+    case evpn-rd {
+      when "derived-from-or-self(/frr-route-map:lib/frr-route-map:route-map/frr-route-map:entry/frr-route-map:match-condition/frr-route-map:condition, 'evpn-rd')";
+      description
+        "Match eVPN route-distinguisher";
+      leaf route-distinguisher {
+        type rt-types:route-distinguisher;
+      }
+    }
+
+    case comm-list-num {
```

Comment:
I think we should do the same thing here as we did with access-lists.
Just use single `comm-list-name` case instead of three different cases.
Even in `frr-bgp-filter.yang` community-lists are addressed by name, not number.

Reply:
Done